### ISSUE TITLE
Hybrid vertical and horizontal wrapper

### ISSUE THREAD

**Issue by kinow** [BSC-ES/autosubmit#292]
In GitLab by @ lbatista on Dec 12, 2017, 12:02

Implement the feature of having a hybrid (vertical and horizontal) wrapper of jobs with groups of sequential jobs that can be executed in parallel.

**Comment by kinow** [comment]
In GitLab by @ lbatista on Dec 12, 2017, 12:06

created branch [`292-hybrid-vertical-and-horizontal-wrapper`](https://earth.bsc.es/gitlab/es/autosubmit/compare/master...292-hybrid-vertical-and-horizontal-wrapper)

**Comment by kinow** [comment]
In GitLab by @ swild on Feb 7, 2019, 16:33

hi @dbeltrankyl 

can you maybe update what the status about the hybrid / vertical-horizontal / horizontal-vertical wrapper is? do they work at all?

see also this issue https://earth.bsc.es/gitlab/es/auto-ecearth3/issues/585 

thanks

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on May 17, 2019, 15:32

Hi @ swild,

Sry the extremely late answer, I really missed this one and I saw it now because I'm checking all the wrappers issues.

As far I know, they are working on 3.12.0b (and 3.12.1b which, will be released today) at least for auto-monarch and horizontal-vertical but should work also vertical-horizontal. This issue has been on the works these months solving multiple issues and won't work properly in the default/stable AS version which is 3.11.0-hotfix.

The main issue that lasts is to work with wrappers and remote dependencies that don't work properly and others bugs in relation with wrappers, like lack of flexibility in some wrappers options that control the number of jobs running at the same time.

I'll close this issue because it is implemented if, you want to use it keep in mind that is still a beta version and bugs can happen and don't hesitate to open a new issue if you found any bug. Also, keep in mind that there is only one project that is using hybrid wrappers, auto-monarch so there is little general feedback.

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on May 17, 2019, 15:32

closed
